pwd_guess.py

```diff
@@ -125,6 +142,11 @@ def encode(self, C, maxlen=None):
         self.encode_into(X, C)
         return X
 
+    def decode(self, X, calc_argmax=True):
```

Comment:
I removed this because its not being used anywhere. It probably crept back in while you were merging. If you think its being used, we can keep it, but otherwise I think this is dead code.